mne/export/_edf.py

```diff
@@ -30,14 +30,19 @@ def _try_to_set_value(header, key, value, channel_index=None):
                            f"{return_val}.")
 
 
-def _export_raw(fname, raw, physical_range):
+def _export_raw(fname, raw, physical_range, add_ch_type):
     """Export Raw objects to EDF files.
 
     TODO: if in future the Info object supports transducer or
     technician information, allow writing those here.
     """
+    # scale to save data in EDF
     phys_dims = 'uV'
 
+    # get EEG-related data in uV
+    units = dict(eeg='uV', ecog='uV', seeg='uV', eog='uV', ecg='uV', emg='uV',
```

Comment:
@cbrnr can you lmk which data channels you would like included here? I'm not a MEG user, so I've only ever used MNE for EEG/iEEG.

Would it be:

```
mag='fT', grad='fT/cm'
```

Anything else?

Reply:
Actually the challenge is more-so on reading in MEG/fnirs data correctly using `read_raw_edf`. I'm not sure how to read in this data correctly. Sure I can easily append "MEG M1" to indicate M1 is a MEG channel, or "FNIRS M2" to indicate M2 is a FNIRS channel, but reading the data back in to do a unit test on the round trip is non-trivial for me.

Is it possible to leave this PR as is and then table it for a future improvement from someone who wants to export MEG/Fnirs to EDF and is familiar with the reader of MEG/Fnirs enough to add that functionality in?

Reply:
+1 for doing it later. I assume YAGNI, and for MEG important information will be lost anyway (at the very least, the head<->device transform, sensor orientations, sensor coil types)

Reply:
The same could be said for EEG as well because EDF doesn't store custom sensor locations. But let's stick to exporting EEG for now, the warning message and code comment are now clear that this is not a restriction imposed by EDF.